src/components/Layouts/Base.tsx

```diff
@@ -17,7 +17,7 @@ const BaseLayout: React.FC<Props> = ({ children, isLoading }) => (
     <Box w="100%">
       <NavBar />
     </Box>
-    <Box w="60%">
```

Comment:
Honestly not a fan of this, I think it's bad practice because it makes stuff like the terms/privacy unreadable. I'd rather we just set a minWidth of like 300px on this or maybe used array syntax for widths - larger % on mobile, smaller on desktop?

Reply:
Also, have realised that Privacy Policy header is centred but Terms isn't - we should change that.